source/jormungandr/jormungandr/stat_manager.py

```diff
@@ -556,7 +564,11 @@ def __call__(self, f):
         def wrapper(*args, **kwargs):
             start_time = time.time()
             call_result = f(*args, **kwargs)
-            self.manager.manage_stat(start_time, call_result)
+            try:
+                self.manager.manage_stat(self, start_time, call_result)
+            except (StatManagerError, Exception):
+                # if stat are not working we don't want jormungandr to stop.
+                pass
```

Comment:
StatManagerError is an Exception, so you may just:
```suggestion
            except Exception:
                # if stat are not working we don't want jormungandr to stop.
                pass
```

Reply:
Done